### PR TITLE
Upgrade_watchdog

### PR DESCRIPTION
**Pull request by nicHoch**

### CODE REVIEW

File: stixcore/version_conf.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ def get_conf_version():
         with open(Path(__file__).parent / "config" / "data" / "common" / "VERSION.TXT") as f:
             return f.readline().strip()
     except Exception:
-        return 'v0.1.3'
+        return 'v0.1.5'
```
Comment (samaloney) [inline stixcore/version_conf.py:9]: Would a place holder be better here `0.0.0` as it likely we won't remember to update this?

File: stixcore/soop/tests/test_soop_manager.py [hunk 2]
```diff
@@ -62,11 +61,11 @@ def test_soop_manager_watchdog(soop_manager):
     time.sleep(3)
 
     # currently not triggered on mac see: https://github.com/i4Ds/STIXCore/issues/149
-    if platform.system() != "Darwin":
-        # the new data should be integrated now
-        assert soop_manager.filecounter == 4
-        assert len(soop_manager.soops) == 129
-        assert len(soop_manager.observations) == 132
+    # if platform.system() != "Darwin":
```
Comment (samaloney) [inline stixcore/soop/tests/test_soop_manager.py:64]: Delete?

File: setup.cfg [hunk 3]
```diff
@@ -28,7 +28,7 @@ install_requires =
     scp
     spiceypy
     sunpy>=4.0.1
-    watchdog
+    watchdog==5.0.3
```
Comment (samaloney) [inline setup.cfg:31]:
```suggestion
    watchdog>=5.0.3
```

File: stixcore/soop/tests/test_soop_manager.py (hunk 2 above)
Comment (samaloney) [inline stixcore/soop/tests/test_soop_manager.py:64]:
```suggestion
```

File: stixcore/version_conf.py (hunk 1 above)
Comment (samaloney) [reply stixcore/version_conf.py:9]: what do you think?

**Review by samaloney — APPROVED** [review]

File: stixcore/soop/tests/test_soop_manager.py (hunk 2 above)
Comment (nicHoch) [reply stixcore/soop/tests/test_soop_manager.py:64]: deleted

File: stixcore/version_conf.py (hunk 1 above)
Comment (nicHoch) [reply stixcore/version_conf.py:9]: integrated

**Pull request merged**